### PR TITLE
Bump testing deps to address jsonwebtoken issues

### PR DESCRIPTION
**Pull request by rlazo**
See

https://deps.dev/maven/com.google.firebase%3Afirebase-firestore/24.6.0

Project side card, then vulnerabilities

### CODE REVIEW

File: smoke-tests/src/androidTest/backend/functions/functions/package.json [hunk 1]
```diff
@@ -10,12 +10,12 @@
     "logs": "firebase functions:log"
   },
   "engines": {
-    "node": "12"
+    "node": "19"
```
Comment (daymxn) [inline smoke-tests/src/androidTest/backend/functions/functions/package.json:13]: I'm assuming this is fine since it's a test?

**Review by daymxn — APPROVED** [review]

**Comment by daymxn** [comment]
This prob deserves a changelog entry btw

File: smoke-tests/src/androidTest/backend/functions/functions/package.json (hunk 1 above)
Comment (rlazo) [reply smoke-tests/src/androidTest/backend/functions/functions/package.json:13]: I think the same

**Comment by rlazo** [comment]
> This prob deserves a changelog entry btw

Since this only affects tests, it doesn't affect devs life

**Pull request merged**